app/controllers/browse_controller.rb

```diff
@@ -1,26 +1,24 @@
 # frozen_string_literal: true
 
 class BrowseController < ApplicationController
-  before_action :normalize_parameters
-
   def index
     @shelf_list = ShelfListPresenter.new(shelf_list_params)
   end
 
   def subjects
-    @subject_list = BrowseList.new(subject_list_params)
+    list = BrowseList.new(subject_list_params)
+    params[:prefix] = list.prefix
```

Comment:
On second thought, let's leave the params hash alone and make the form use `@list.prefix`. I think forcibly changing the params is kinda bad form (literally) anyway.